tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
Yeah, I'm fine with defaulting to package install, but adding a note in the README about current LTS distros that will require install from source.

I just want to make sure we have the tests for each supported major LTS release (centos6, centos7, ubuntu 14/16, debian 8).